packages/marko-web/src/components/elements/core/image.marko

```diff
@@ -8,6 +8,11 @@ $ let src = result == null ? '' : `${result}`.trim();
 $ src = /^http/i.test(src) ? src : null;
 $ const collapse = shouldCollapse(input.collapse);
 
+$ if (input.obj.isLogo) {
```

Comment:
No deep object access please 🙂
Also, allow an input.logo to set this regardless of what isLogo is to